_docs/addons/sked/config.md

```diff
@@ -327,7 +327,7 @@ if (rex::isBackend() && rex_get('sked_cat_id') && rex::getUser()) {
 ```
 
 <a name="yform"></a>
-## yform
+## YForm
 
 In yform muss noch die Kategorientabelle `rex_sked_event_categories` über den Tablemanager angelegt werden. In unserem Falle benötigen wir lediglich die Felder `name_1` und `name_2`, also die Namen für die Sprache 1 und Sprache 2.
```

Comment:
YForm-Schreibweise hier auch vereinheitlichen.

Reply:
ggf. sonst überpüfen.

Reply:
Wie meinst du vereinheitlichen?, überall yform mit YForm anpassen?.

Reply:
In der Zeile drunter steht z.B. wieder `yform` -  das korrigieren.

Reply:
Gemacht.